### DISCUSSION TITLE
FriendlyElec NanoPi NEO 无法启动

### DISCUSSION THREAD

**Discussion by excpu** [coolsnowwolf/lede#12065] [Bugs / 问题反馈]
Ubuntu LTS 下编译一遍过，但是刷写进sd卡无法启动，网口灯常亮，有人知道为何吗？

**Comment by Gabrielxzx** [comment]
nanopi R2S 和 orangepi_zero 也无法启动，没TTL线，暂时也没办法抓LOG

**Comment by lysgwl** [comment]
遇到同样的问题，编译了好几个版本，r2s设备上电后，就是无法启动，估计是uboot的问题。
